### PR TITLE
Fix setting boolean variable

### PR DESCRIPTION
**Pull request by digeff**
Fix needed for this test: https://github.com/microsoft/vscode-chrome-debug/pull/870

Setting a boolean was throwing the exception on line 173 before this change

### CODE REVIEW

File: src/chrome/chromeUtils.ts [hunk 1]
```diff
@@ -170,7 +172,7 @@ export function remoteObjectToValue(object: CDTP.Runtime.RemoteObject, stringify
             }
         } else {
             if (object.description === undefined) {
-                throw new Error(`Expected an object that is neither objecr, not function nor undefined to have a description yet it didn't: ${JSON.stringify(object)}`);
+                throw new Error(`Expected an object that is neither object, not function nor undefined to have a description yet it didn't: ${JSON.stringify(object)}`);
             }
 
             // The value is a primitive value, or something that has a description (not object, primitive, or undefined). And force to be string
```
Comment (roblourens) [inline src/chrome/chromeUtils.ts]: I thought it was supposed to be handled by this block

Comment (roblourens) [reply src/chrome/chromeUtils.ts]: Are you sure the exception is correct? I guess that isn't in master

Comment (digeff) [reply src/chrome/chromeUtils.ts]: I missed that before. Changing implementation

**Comment by roblourens** [comment]
Yeah this is basically what is in master, thanks

**Pull request merged**